app/components/drafts/drafts.js

```diff
@@ -49,7 +49,7 @@ export default ({ p2p, profile }) => {
               <>
                 {modules.length ? (
                   'You’ve reached the end! ✌️'
-                ) : hasListedContent ? (
+                ) : hasOnlyListedContent ? (
```

Comment:
I see now why you named that variable `hasListedContent` originally, really it is `hasListedAndOnlyListedContent`, right?

```suggestion
                ) : profile.rawJSON.contents.length && hasOnlyListedContent ? (
```